internal/util/sync/updatefile.go

```diff
@@ -53,7 +53,10 @@ func SetDependency(dependency kptfile.Dependency) error {
 	// add the dependency
 	if !found {
 		if dependency.Strategy == "" {
-			dependency.Strategy = string(update.FastForward)
+			dependency.Strategy = string(update.KResourceMerge)
```

Comment:
do you want to rename this package from 'sync'?

Reply:
Will do it as part of merging `sync` with `update` in coming PRs.